### PR TITLE
Pull Request for "Branch: Desktop Version #1"

### PR DESCRIPTION
**Pull request by DebbyMiressa**
The branch adds media query to the style.css in order to incorporate several screen size support.
- Affected files: Index.html, style.css, and README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,19 +20,26 @@
...
+            <a class="cont" href="#">Contact</a>
+            <a class="mail"><img src="images/Mail.png" alt="mail"></a>
+          </div>
           <button id="menu" class="button">
             <img src="images/Menu@1x.png" alt="Header" />
           </button>
         </header>
 
         <section class="headline">
+          <div class="main-headline">
```
Comment (muneebulrehman) [inline index.html:35]: The icons are not aligned as per the Figma template. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -55,13 +63,19 @@ <h2 id="supportingtext">I'm a Software Developer</h2>
       </div>
       <div>
         <section class="recentworks">
+          <div class="zero">
```
Comment (muneebulrehman) [inline index.html:66]: Also, your headline work section is not matching the Figma template. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -55,13 +63,19 @@ <h2 id="supportingtext">I'm a Software Developer</h2>
       </div>
       <div>
         <section class="recentworks">
```
Comment (muneebulrehman) [inline index.html:65]: Your work section layout is not matching the Figma template on the larger screens. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -144,17 +159,20 @@ <h1 class="aboutme">About me</h1>
           <div class="endline">
             <img src="images/endline.png" alt="line" />
           </div>
-          <ul class="lfs">
-            <li>
+          <div class="deskendline">
+            <img src="images/deskendline.png" alt="deskendline">
+          </div>
+          <div class="lfs">
```
Comment (muneebulrehman) [inline index.html:165]: Your languages section is also not aligned properly for larger screens. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -173,22 +191,27 @@ <h1 class="aboutme">About me</h1>
...
                 <li>Terminal</li>
               </ul>
-            </li>
-          </ul>
+            </div>
+          </div>
         </section>
       </div>
     </div>
     <div class="footer">
```
Comment (muneebulrehman) [inline index.html:199]: Your footer layout is also breaking for larger screens. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @DebbyMiressa , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @DebbyMiressa 

Great work on making the changes requested by a previous reviewer 👏 🎉
 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so close

## Highlights
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

## Required Changes ♻️

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DebbyMiressa ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Pull request ✔️
- Linters are passing ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**